apps/site/src/content/blog/state-of-the-budgie-2024.mdx

```diff
@@ -0,0 +1,273 @@
+
+---
+
+### 2024 In Retrospect
+
+As you can see, there was a massive amount of work that happened over the course of 2024 towards our goal of moving to Wayland. As you may have noticed, we did not end up delivering Budgie 10.10 before end of year. As is the classic case of software development, some aspects of the experience have been proven to be trickier than anticipated, others requiring more work, some originally unscoped / undefined items needing to be worked on (labwc bridge as an example) to improve usability, and more.
+
+If we put it all into perspective though, we accomplished the massive goal of Budgie Desktop being daily-drivable on Wayland for a typical end user usecase. You are able to:
+
+- Log into a Budgie Desktop session on top of labwc from your login greeter of your choice
+- Launch applications and they're grouped as you would expect
+- Open up Budgie Menu, Raven, various popovers and they are correctly positioned.
+- Switch up your panel position.
+- Assuming you have the wlr and gtk xdg-desktop-portals installed, share your screen in applications.
+- Livestream using OBS Studio with screen capture working as expected
+- Once the labwc bridge PR lands, change many settings in Budgie Control Center and have them change the behavior of the compositor.
+
+At the start of the 2024, none of this was possible. All the code that leveraged libxfce4windowing was untested under Wayland.
+
+There is still some work to be done to deliver Budgie 10.10, but let's dive into this year to find out about that!
+
+---
+
+## 2025
+
+Similar to our epics for last year, this year's are centered around Budgie 10 and 11.
+
+### First Epic: Ship Budgie 10.10
+
+There are still some key areas that need improving or fixing before we can ship Budgie 10.10 to end users.
+
+#### TODO
+
+For items within the budgie-desktop repo itself, we have updated our [10.10 milestone](https://github.com/BuddiesOfBudgie/budgie-desktop/milestone/13) with the related items.
+
+**Applets**
+
+Some applets need to be updated to reach complete or some degree of feature parity with their usage under the "legacy" X11 environment.
+
+- Keyboard Layout applet: This currently uses IBus and `Gnome.Languages` and needs to be updated so keyboard layout changing via the applet works.
+- Night Light applet: This needs to be updated to leverage the [wlr gamma control protocol](https://wayland.app/protocols/wlr-gamma-control-unstable-v1). It will most likely do this by effectively being an indicator similar to gammastep-indicator, leveraging [gammastep](https://gitlab.com/chinstrap/gammastep) for starting / stopping gammastep and setting a specific temperature. Budgie Daemon v2 will eventually handle gamma setting for us.
+- Tasklist applet: Not to be confused with IconTasklist, this "legacy" tasklist provides a horizontal list of running apps, ungrouped, similar to `Wnck.Tasklist`. Evan has started rewriting this however some more work needs to be done on this front.
```

Comment:
I actually didn't start Simple Tasklist, I just took over the work after the original person stopped.

Reply:
True

Reply:
Should be resolved in ed01480